source/administration/plugins.rst

```diff
@@ -252,7 +266,11 @@ To manage plugins, go to **System Console > Plugins > Plugin Management**. From
 
 Pre-packaged Plugins
 ~~~~~~~~~~~~~~~~~~~
-Mattermost ships with a number of pre-packaged plugins written and maintained by Mattermost. Instead of building these features directly into the product, you can selectively enable the functionality your installation requires. Pre-packaged plugins cannot be removed via the System Console, but can be customized by modifying the ``prepackaged_plugins`` directory in your Mattermost installation.
+Mattermost ships with a number of pre-packaged plugins written and maintained by Mattermost. Instead of building these features directly into the product, you can selectively enable the functionality your installation requires. Pre-packaged plugins can be removed after 5.20 via the System Console, or by modifying the ``prepackaged_plugins`` directory in your Mattermost installation.
+
+As of 5.20 Mattermost now ships pre-downloaded plugins that will be visible in the PLugin marketplace even if your Mattermost server does not have external internat acess as in some secure settings.  Only Mattermost "official" plugins are shipped with the Mattermost package as a result, community plugins are not available in the Plugin Marketplace on servers without internet access.
+
+If you previously had some plugins that were pre-packaged on your server, but were not enabled at time of upgrading to 5.20+, they will no longer be installed your server.  As mentioned - those previously pre-packaged plugins will still be available to be installed on your server via the Plugin Marketplace. 
```

Comment:
If we simplify as above, we can simplify this further to emphasize the positive: "Previously enabled, pre-packaged plugins will remain installed on upgrading to v5.20. Any disabled, pre-packaged plugins will not show up in the system console but may be installed from the plugin marketplace."